### ISSUE TITLE
Error con build project 

### ISSUE THREAD

**Issue by jacintaherrera** [PUC-IIC3413/2022-1-MilleniumDB#1]
Hola hace dias tengo problemas al ejecutar "cmake -Bbuild/Release -DCMAKE_BUILD_TYPE=Release && cmake --build build/Release/".
<img width="695" alt="Screen Shot 2022-04-05 at 19 46 05" src="https://user-images.githubusercontent.com/72674143/161869184-aa99acc3-f0db-4113-a8a7-dd289ab17c3e.png">
<img width="695" alt="Screen Shot 2022-04-05 at 19 46 17" src="https://user-images.githubusercontent.com/72674143/161869190-061c55b6-033b-453c-a0e1-dc4003578401.png">
me entrega esos errores al correr el código anterior. 
Además me dice que  Cannot find "usr/include"  que es lo que aparece en el archivo c_cpp_properties.json. 
<img width="329" alt="Screen Shot 2022-04-05 at 19 48 18" src="https://user-images.githubusercontent.com/72674143/161869426-2ef5a28d-09d2-4595-a63d-62012b72f514.png">

tengo Mac y no es primera vez que tengo problemas con C++. Nose como arreglar estos errores :(

**Comment by cirojas** [comment]
¿Estás usando una máquina virtual de linux? ¿Qué distribución?

El problema de la imagen parece ser con la librería boost. No todas las versiones funcionan como debería. He visto que funciona bien con las versiones 1.71, 1,74 y 1.76; y que falla con la 1.78.

**Comment by jacintaherrera** [comment]
No, estoy usando directamente Mac. Ahora use la version 1.76 y me salió esto
<img width="1227" alt="Screen Shot 2022-04-06 at 12 14 33" src="https://user-images.githubusercontent.com/72674143/162020742-fbf037f5-bb1b-4ae1-b3fb-0da14a5cad6b.png">

**Comment by cirojas** [comment]
En la línea 98 de src/query_optimizer/quad_model/join_order/leapfrog_optimizer.cc:
`auto enumeration_level = var_order.size();`
Reemplazar por:
`uint_fast32_t enumeration_level = var_order.size();`

Yo recomiendo usar una máquina virtual con ubuntu 20.04, en mac van a seguir saliendo errores probablemente, y no se si todos se podrán arreglar fácilmente como en este caso
